docs/src/modeler_guide/enumerated_types.md

```diff
@@ -0,0 +1,100 @@
+
+# Enumerated Types
+
+Some of the fields of `Component` structs are specified with `Enum` types. This is useful
```

Comment:
You might want to refer to the fact that we are using derived `scoped_enums` instead of regular Julia enums and give an example.
```
gen.fuel = ThermalFuels.COAL
```